scripts/masternode.js

```diff
@@ -61,14 +62,18 @@ export default class Masternode {
         return cMasternode ? cMasternode.status : 'MISSING';
     }
 
+    /**
+     * @param {String} ip
```

Comment:
Add the `port` param to the JSDocs
```suggestion
     * @param {String} ip
     * @param {Number} port
```